app/src/main/java/com/springboot/webpush/controller/model/WebPushMessage.java

```diff
@@ -1,12 +1,26 @@
 package com.springboot.webpush.controller.model;
 
-import com.springboot.webpush.common.api.PushMessage;
+import org.springframework.util.StringUtils;
+
+import com.springboot.webpush.common.api.types.OnActionClickOperation;
 
 import nl.martijndwars.webpush.Urgency;
 
+/**
+ * The external {@link WebPushMessage} contains the data required to generate a custom web notification.
+ */
 public class WebPushMessage {
-    private Urgency urgency;
-    private String  message;
+    @Deprecated
```

Comment:
Do you really need this deprecated? Just delete it?

Reply:
I wanted to delete it, but I need you to make the corresponding change on the UI!!!  If you can do that. I will delete it. Thanks

Reply:
FYI -- I deleted it -- you need to make the FE change -- Added it to the other SMALL PR.